dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts

```diff
@@ -142,12 +142,12 @@ export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
         to: { name: 'verifications' },
         disabled: !this.$store.state.twin?.isOwner,
       },
-      {
+      ...(this.$store.state.twin.isOwner ? [{
```

Comment:
This seems very hard to read and not declarative at all. I would prefer an explicit filter either in this function, or, prolly better, in the created hook, where it is called :)

Reply:
yeah it's kinda hard to read, but there is not real a useful key where to filter for, and I really don't want to go with array indices. I also think it makes no sense to add an item when we already know, that we remove it.

Shall I add a comment for this?

Reply:
I think it's better to just make it invisible in the template instead. It's UI related and should be handled there instead of cluttering the data model. It's cleaner that way.

Reply:
Hm, I'm not satisfied with this really, but it works for now :D